TCAT/AppDelegate.swift

```diff
@@ -24,6 +24,27 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         
+        // Update shortcut items
+        var shortcutItems = [UIApplicationShortcutItem]()
+        let favorites = SearchTableViewManager.shared.retrieveRecentPlaces(for: Constants.UserDefaults.favorites)
+        for itemType in favorites {
+            switch itemType {
+            case .busStop(let busStop):
+                let data = NSKeyedArchiver.archivedData(withRootObject: busStop)
+                let placeInfo: [AnyHashable: Any] = ["place": data]
+                let shortcutItem = UIApplicationShortcutItem(type: busStop.name, localizedTitle: busStop.name, localizedSubtitle: nil, icon: UIApplicationShortcutIcon(type: .location), userInfo: placeInfo)
```

Comment:
think u mentioned this at the meeting but d4get to add a star icon instead of location icon